src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1287,6 +1290,12 @@ rawInlineOr name' fallback = do
      then rawInline "latex" <$> getRawCommand name' ("\\" <> name')
      else fallback
 
+dropLineBreaks :: Inlines -> Inlines
+dropLineBreaks = Many . Data.Sequence.filter isNotLineBreak . unMany
+    where
```

Comment:
Do you want to remove only line breaks at the top level, or nested ones too? (e.g. a LineBreak inside an Emph)

Reply:
Thanks for pointing this out. I missed this case. I **do** want to remove the nested line breaks. For example, see the third line in the screenshot I sent to the previous conversation.